assignments/lambda-classes.js

```diff
@@ -1 +1,128 @@
+    }
+    demo(subject){
+        return `Today we are learning about ${subject}`;
+    }
+    grade(student, subject){
+        return `${student.name} receives a perfect score on ${subject}`;
+    }
+}
+
+
+// -----------------------------------------------
+
+// #### Student
+
+// * Now we need some students!
+// * Student uses the same attributes that have been set up by Person
+// * Student has the following unique props:
+//   * `previousBackground` i.e. what the Student used to do before Lambda School
+//   * `className` i.e. CS132
+//   * `favSubjects`. i.e. an array of the student's favorite subjects ['Html', 'CSS', 'JavaScript']
+// * Student has the following methods:
+//   * `listsSubjects` a method that logs out all of the student's favoriteSubjects one by one.
+//   * `PRAssignment` a method that receives a subject as an argument and logs out that the `student.name has submitted a PR for {subject}`
+//   * `sprintChallenge` similar to PRAssignment but logs out `student.name has begun sprint challenge on {subject}`
+
+class Student extends Person{
+    constructor(attribs){
+        this.previousBackground = attribs.previousBackground,
+        this.className  = attribs.className,
+        this.favSubjects = attribs.favSubjects,
+    }
+    listsSubjects(arr){
+        return arr.forEach((e) => console.log(e));
+    }
+    prAssignment(subj){
+        console.log(`${this.name} has submitted a PR for ${subj}`);
+    }
+    sprintChallenge(sub){
+        console.log(`${this.name} has begun sprint challege on ${subj}`);
+    }
+}
+
+// -----------------------------------------------
+
+// #### Project Mananger
+
+// * Now that we have instructors and students, we'd be nowhere without our PM's
+// * ProjectManagers are extensions of Instructors
+// * ProjectManagers have the following uniqe props:
+//   * `gradClassName`: i.e. CS1
+//   * `favInstructor`: i.e. Sean
+// * ProjectManangers have the following Methods:
+//   * `standUp` a method that takes in a slack channel and logs `{name} announces to {channel}, @channel standy times!​​​​​
+//   * `debugsCode` a method that takes in a student object and a subject and logs out `{name} debugs {student.name}'s code on {subject}`
+
+class ProjectManager extends Instructor{
+    constructor(attribute){
+        super(attribute),
+        this.gradClassName = attribute.gradClassName,
+        this.favInstructor = attribute.favInstructor, 
+    }
+    standUp(channel){
+        console.log( `${this.name} announces to ${channel}, @${channel} standy times!​​​`​​);
+    }
+    debugsCode(student, subject){
+        console.log(`${this.name} debugs ${student.name}'s code on ${subject}`)
+    }
+
+
+
+
+// -----------------------------------------------
+
+
+const fred = new Instructor({
+    name: 'Fred',
+    location: 'Bedrock',
+    age: 37,
+    gender: 'male',
+    favLanguage: 'JavaScript',
+    specialty: 'Front-end',
+    catchPhrase: `Don't forget the homies`
+  });
+
+
```

Comment:
looks like you're missing the rest of the MVP here, make sure to go through the readme and see what else you need to do